puzzlejs/puzzle.js

```diff
@@ -1866,6 +1935,10 @@ function PuzzleGrid(puzzleEntry, options, container, puzzleId) {
             if (this.screenreaderSupported) { td.role = "cell"; td.ariaColIndex = this.leftClueDepth + c + 1; }
             td.classList.add("cell");
             td.classList.add("inner-cell");
+            if (this.options["data-text-avoid-position"] == "top")
```

Comment:
Similar to the prior comment on positions: `let avoid = this.options["data-text-avoid-position"]; if (avoid) { td.classList.add("pos-avoid-" + avoid); }`